### ISSUE TITLE
Missing tabs translation languages when you reopen the program

### ISSUE THREAD

**Issue by tom-vb** [crow-translate/crow-translate#398]
**Bug description**
Sometimes, not always, especially if the program does not close well, either by restarting the computer or by whatever the languages you had in the translation tabs disappear, are not saved and when you reopen the program are not there

**Steps to reproduce**
restart de computer without closing the program

-->

**Expected behavior**
that when I open the program are there the tabs of the languages that I had open the last time I used it

**Screenshots**

**Environment**

- OS: windows 10 ltsb

- Crow Translate version: 2.9.1

**Comment by Shatur** [comment]
This can happen if the program was terminated abnormally. The program saves it state on exit. You most likely terminated the program or did a force shutdown. I don't have this issue on my Windows machine.

**Comment by tom-vb** [comment]
Hi!, thanks for your response. Perhaps it would be better solution to to save it every time a change is made and not when exiting the program. It becomes annoying if, for whatever reason, the program has not been able to close correctly, having to enter all the input and output languages ​​again.

**Comment by Shatur** [comment]
> Perhaps it would be better solution to to save it every time a change is made and not when exiting the program.

The problem is that I will have to write data to disk very often. So I haven't decided yet.

**Comment by tom-vb** [comment]
It is your choice. I only make suggestions as a user who is delighted to have found this application.

**Comment by pqnhan** [comment]
Well, I ran into this problem on my Ubuntu as well.
 Every time I shutdown the computer in an unusual way (rebooted, forgot to close the Crow program window), Crow's language settings are reset.

 It would be better if the configuration file was permanently saved on the hard drive every time I change the program's configuration settings and it would automatically reload every time the program was opened, regardless of the last time the program was closed in the normal way or not.
